config/routes.rb

```diff
@@ -36,4 +38,13 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  post "/applications", to: "applications#create"
+  get "/applications/new", to: "applications#new"
+  get "/applications/:id", to: "applications#show"
+  post "/applications/:id", to: "pet_applications#create"
+  patch "/applications/:id", to: "applications#update"
+
+  get "/admin/applications/:id", to: "admin_applications#show"
```

Comment:
Just a note here (line 48) its okay to have your controller called `admin_applications` but another way that you could build this is to have a directory called `admin` and within that directory have a controller called `applications`. So something like:
```ruby
controllers/admin/applications_controller.rb
```